rmw_fastrtps_shared_cpp/src/qos.cpp

```diff
@@ -160,11 +160,8 @@ get_datawriter_qos(
 bool
 is_valid_qos(const rmw_qos_profile_t & qos_policies)
 {
-  if (qos_policies.liveliness == RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE ||
-    qos_policies.liveliness == RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC ||
-    !is_time_default(qos_policies.liveliness_lease_duration))
-  {
-    RMW_SET_ERROR_MSG("Liveliness QoS is not yet supported for fastrtps.");
+  if (qos_policies.liveliness == RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE) {
+    RMW_SET_ERROR_MSG("Liveliness policy MANUAL_BY_NODE is not yet supported for fastrtps.");
```

Comment:
We should ask if `MANUAL_BY_NODE` is actually supported and the missing `assert_liveliness()` API is just an oversight, because `MANUAL_BY_TOPIC` is supposed to be the stricter policy and that's supposedly supported.

Reply:
Asked on https://github.com/eProsima/Fast-RTPS/pull/574